web/web.go

```diff
@@ -29,8 +29,8 @@ import (
 
 // Commandline flags.
 var (
-	listenAddress  = flag.String("listenAddress", ":9090", "Address to listen on for web interface.")
-	useLocalAssets = flag.Bool("useLocalAssets", false, "Read assets/templates from file instead of binary.")
+	listenAddress  = flag.String("web.listen-address", ":9090", "Address to listen on for the web interface and API.")
```

Comment:
If we're going to be changing this flag name I think we should also ensure all the default ports are unique across projects while we're at it.